### DISCUSSION TITLE
Trouble to use Auth in local

### DISCUSSION THREAD

**Discussion by binajmen** [supabase/supabase#4679] [Questions] [answered]
Hello,

I'm playing with the local environment of Supabase. Everything looks good and is running, but I can't figure out how I can use the Auth part.

I've created a user, but impossible to send a password recovery or send a magic link. In the studio, I receive a green notification but nothing happen.

I did not find documentation on how to setup the Auth part in local (smtp config? dumb password?).

Any help would be greatly appreciated 🙏

**Comment by J0** [comment, marked as answer]
Hey,

 You will need to populate the [Docker config](https://github.com/supabase/supabase/blob/master/docker/.env.example#L49) with the corresponding [GoTrue env vars](https://github.com/supabase/gotrue/blob/master/example.env#L4)

Let me know if there are further questions.

Thanks!
